blockchain/state/statedb.go

```diff
@@ -326,6 +326,18 @@ func (self *StateDB) IsValidCodeFormat(addr common.Address) bool {
 	return false
 }
 
+func (self *StateDB) GetCodeFormat(addr common.Address) params.CodeFormat {
```

Comment:
I think we need to add comments remarking that the method can return an invalid code format, `params.CodeFormatLast` and it can be validated with `CodeFormat.Validate()`.

Reply:
Thanks! Reflected at 512195885257c4ea89242bdbe39a76c626584a92